Makefile

```diff
@@ -5,13 +5,24 @@ export GIT_DIFF ?= off
 .PHONY: help lint lint-fix build test-unit test-integration test-integration-full deploy clean clean-deploy
 
 help:
-	echo "Please choose one of the following targets: lint, lint-fix, build, test-unit, test-integration, test-integration-full, deploy, clean, clean-deploy"
+	echo "Please choose one of the following targets: lint, lint-fix, build, test-unit, test-integration, test-integration-dry, deploy, clean, clean-deploy, serialize_package"
```

Comment:
[Here the same](https://github.com/CartoDB/carto-spatial-extension/pull/126/files#r672871164). We should also follow the naming convention here: `serialize-package`, but I think `build-package` should be more convenient

Reply:
Having the advanced PR context I think `serialize-package` is correct